docs/product-user-guide/src/main/asciidoc/decision-tables-rule-table-entries-ref.adoc

```diff
@@ -0,0 +1,87 @@
+
+The following table lists the supported labels (column headers) and values for `RuleTable` definitions. For column headers, you can use either the given labels or any custom labels that begin with the letters listed in the table.
+
+*<@Michael (4 of 6): I've done my best to make the `Initial` column (now custom label column) more intuitive, but I think this feature is pretty weird and might do more harm that help. Is there a good reason why it's worth documenting it, rather than just ditching it and having users simply use the standard labels? Both here and in the attributes table, but oddly not in the RulSet table. It's not consistent and seems unhelpful. Thoughts?>*
+
+.Supported `RuleTable` definitions
+[cols="25%,15%,35%,25%", options="header"]
+|===
+|Label |Or custom label that begins with |Value |Usage
+
+|`NAME`
+|N
+|Provides the name for the rule generated from that row. The default is constructed from the text following the `RuleTable` tag and the row number.
+|At most one column.
+
+|`DESCRIPTION`
+|I
+|Results in a comment within the generated rule.
+|At most one column.
+
+|`CONDITION`
+|C
+|Code snippet and interpolated values for constructing a constraint within a pattern in a condition.
+|At least one per rule table.
+
+|`ACTION`
+|A
+|Code snippet and interpolated values for constructing an action for the consequence of the rule.
+|At least one per rule table.
+
+|`METADATA`
+|@
+|Code snippet and interpolated values for constructing a metadata entry for the rule.
+|Optional, any number of columns.
+|===
+
+// The cells that follow condition, action, or metadata attributes are used together to form the pattern for constraints, consequences, or annotations for the rules in the decision table.
+
+The following sections provide more details about condition, action, and metadata columns.
+
+Conditions::
+
+For columns headed `CONDITION`, the cells in successive lines result in a conditional element:
+
+* *First cell:* Text in the first cell below `CONDITION` develops into a pattern for the rule condition, and uses the snippet in the next line as a constraint. If the cell is merged with one or more neighboring cells, a single pattern with multiple constraints is formed. All constraints are combined into a parenthesized list and appended to the text in this cell. If the cell is left blank, the code snippet in the next row must result in a valid conditional element on its own.
++
+To include a pattern without constraints, you can write the pattern in front of the text of another pattern, with or without an empty pair of parentheses. You can also append a `from` clause to the pattern.
++
+If the pattern ends with `eval`, code snippets produce boolean expressions for inclusion into a pair of parentheses after `eval`.
++
+*<@Michael (5 of 6): Isn't this first cell also optional, like it is with Actions below? And shouldn't we say something similar to what we say for the first cell in the Actions section below?>*
```

Comment:
@sterobin I'd be inclined to remove (re-word) https://github.com/sterobin/kie-docs/pull/10/files#diff-82b1ecd3589462e9deb4a8011f6f3ba4R21 and https://github.com/sterobin/kie-docs/pull/10/files#diff-82b1ecd3589462e9deb4a8011f6f3ba4R22 as what is written there only applies to CONDITION columns and you state the same, but in more detail, here. Perhaps the first occurrences I link need to state the use of the two rows in between "Rule attributes" and "Column label" depend upon the type of Rule (column) Attribute?

Reply:
@sterobin My comments above (your "(1 of 6)") should be integrated into the text here. I think that'd clarify it.